mkdocs.yaml

```diff
@@ -71,6 +71,9 @@ markdown_extensions:
       format: svg
   - pymdownx.arithmatex
   - pymdownx.highlight
+  - pymdownx.snippets:
+      auto_append:
+        - includes/abbreviations.md
```

Comment:
Okay, I'll investigate the usage and fix the issue!

Reply:
@kenji-miyake I think the autofix commit is the culprit here. 

In the [mkdocs documentation for tooltips](https://squidfunk.github.io/mkdocs-material/reference/tooltips/#adding-a-glossary), each entry in the abbreviations file should have an asterisk at the start of the line. However the [ci(pre-commit): autofix](https://github.com/autowarefoundation/autoware-documentation/pull/150/commits/ab2e0d10478b1c7eff89b128225842c1ab199b70) has replaced the asterisks with underscores.

In theory, reverting that change should fix the issue. 🤔

Reply:
I see. I'll check it.

Reply:
Hmm, if I revert the pre-commit changes, it didn't work in my local machine. :thinking:

Reply:
@LalithVipulananthan I'm sorry to be late.

You needed the `abbr` extension.
Also, as you say, `_` instead of `*` didn't work. So I added `prettier-ignore`.

Now it works!
![image](https://user-images.githubusercontent.com/31987104/175835084-1769991a-9883-40a9-ae4d-445699beef80.png)